scipy/sparse/csgraph/_flow.pyx

```diff
@@ -45,6 +45,14 @@ def maximum_flow(csgraph, source, sink):
         The source vertex from which the flow flows.
     sink : int
         The sink vertex to which the flow flows.
+    method: str
+        The method/algorithm to be used for computing the maximum flow.
+        Following methods are supported,
+
+            * 'edmonds_karp': Edmonds Karp algorithm in [1].
+            * 'dinic': Dinic's algorithm in [4].
+
+        Optional, by default, 'edmonds_karp'.
```

Comment:
Probably adding a `versionadded` makes sense for the introduction of `'dinic'` as a `method`.

Reply:
I have added. Let me know if it's the correct way. Thank you.